resources/metadata/artifactPrepareVersion.yaml

```diff
@@ -357,3 +357,9 @@ spec:
           params:
             - name: buildTool
               value: maven
+    - image: gradle:6-jdk11-alpine
```

Comment:
Are we sure this includes everything we need? In the past alpine image have been very lean and missing basic tools like curl or wget that we needed by that time.

Reply:
The image has git, wget, java, gradle... I tested and the artifactPrepareVersion with versioningType: cloud creates a new tag. Works fine. 

The artifactPrepareVersion step uses go-packages for pushing changes to GH, and seems like needs only Java to be installed (when buildTools is gradle), because the step runs ./gradlew (Gradle start up script for UN*X) (the file should be in the repository), gradlew script checks java, and downloads gradle (URL is determined here: https://github.tools.sap/project-piper/azure-demo-k8s-gradle/blob/54b3edb83b149a2e2745e43c3378c121218fe5a3/gradle/wrapper/gradle-wrapper.properties#L3).